brave/src/test/java/com/linecorp/armeria/it/brave/BraveIntegrationTest.java

```diff
@@ -416,7 +416,8 @@ public void testClientTimesOut() throws Exception {
 
         // Client timed out, so no response data was ever sent from the server. There is no wire send in the
         // server and no wire receive in the client.
-        assertThat(serverSpan.annotations()).hasSize(1);
+        // FIXME(ikhoon) the server receives RST_STREAM frame, does it make sence?
+        assertThat(serverSpan.annotations()).hasSize(2);
```

Comment:
I wonder why this has changed. 🧐

Reply:
I think it makes sense because the client would reset the stream when aborting the request. Could we add some comments about this and add more assertions for the properties of `serverSpan.annotations().get(1)` instead?

I also wonder what happens for HTTP/1, too. Could you check?

Reply:
Sure, I will check HTTP/1 and add more assertions!